internal/api/deployments.go

```diff
@@ -50,10 +50,10 @@ type DeploymentCreate struct {
 	Description            string                 `json:"description,omitempty"`
 	EnforceParameterSchema bool                   `json:"enforce_parameter_schema,omitempty"`
 	Entrypoint             string                 `json:"entrypoint,omitempty"`
-	FlowID                 uuid.UUID              `json:"flow_id"`
+	FlowID                 uuid.UUID              `json:"flow_id"` // required
```

Comment:
There's no `required` option (that I'm aware of), so I added `// required` here to signify that we're intentionally not specifying `omitempty`. My thought process was this may help us in the future to determine whether the field was left off intentionally or not - open to opinions though